molecule/default/tests/test_default.py

```diff
@@ -1,14 +1,51 @@
 import os
-
+import pytest
 import testinfra.utils.ansible_runner
 
+
+"""ASC-240: Verify the requested glance images were uploaded
+
+RPC 10+ manual test 10
+"""
+
 testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
-    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('all')
+    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('compute-infra_hosts')[:1]
+
+# attach the utility container:
+attach_utility_container = "lxc-attach -n `lxc-ls -1 | grep utility | head -n 1` -- bash -c "
+
 
+@pytest.mark.jira('asc-240')
+@pytest.mark.xfail(reason='expected to fail until service setup is successfully completed')
```

Comment:
I don't understand why this is expected to fail. What service setup is necessary to run `openstack` commands on the utility container?

Reply:
@johnduarte xfail will not fail the test if it is unexpected passed (like issue has been fixed), if the test passes, it will show `xpassed` in the results and if the test failed (as expected) it will show xfail. Both case won't impact the whole test run report.

I put xfail in the test for the first few test runs because it's heavily depending on the service setup in tasks/main.yml. I am not sure if the service setup is successful or not. I'll keep an eye on it and when everything is good, xpassed can be seen and I will remove the xfail pytest.mark.

We experienced network creating issue in neutron-deploy for several weeks so the network tests always fail and cause the CI red for a while. This pytest.mark.xfail will help to separate the setup issue from the test failing issue